gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
I couldn't find rspec doc but here is a [SO link](http://stackoverflow.com/questions/21544519/correct-way-to-test-lib-folder-with-rspec):
rspec should set up `$LOAD_PATH`